sphinxlint/__main__.py

```diff
@@ -109,6 +117,16 @@ def __call__(self, parser, namespace, values, option_string=None):
         help="comma-separated list of fields used to sort errors by. Available "
         f"fields are: {SortField.as_supported_options()}",
     )
+    parser.add_argument(
+        "-j",
+        "--jobs",
+        metavar="N",
+        action=StoreNumJobsAction,
+        help="Run in parallle with N processes, defaults to \"auto\". "
+        "Special value \"auto\" will set N to cpu-count. "
```

Comment:
Beautiful is better than ugly (and escaped quotes are in the latter group!)
```suggestion
        help="Run in parallle with N processes, defaults to 'auto', "
        "which sets N to the number of logical CPUs."
```